documentation/slate/source/includes/disruptions.md

```diff
@@ -0,0 +1,60 @@
+Disruptions
+===========
+
+
+In this paragraph, we will explain how the disruptions are displayed in the different APIs.
+
+# Context
+
+## periods
+
+### production period of coverage data
+The production period is global to a navitia coverage and is the validity period of the coverage's data.
+This production period cannot exceed one year.
+
+### publication period of a disruption
+The publication period of a disruption is the period on which we want to display the disruption in navitia.
+
+The creator of the disruption might not want the traveller to know of a disruption before a certain date (because it's too uncertain, secret, ...)
+The publication period is the way to control this.
+
+### application periods of a disruption
+The application periods are the list of periods on which the disruption is active.
```

Comment:
Don't know if you want to follow this comment (repeating because github cropped it).
We may add that `For example it's the actual period when railworks are done and train circulation is cut.`